Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18-alpine3.15 as base
+FROM node:20-alpine as base
```

Comment:
Might want to add a specific version here but not necessary - leave it to your decision

Reply:
Added 20-alpine3.18 version